tests/acceptance/new-test.js

```diff
@@ -331,4 +331,36 @@ describe('Acceptance: ember new', function() {
     expect(pkgJson.name).to.equal('foo', 'uses addon name for package name');
   }));
 
+  it('ember new adds ember-welcome-page by default', co.wrap(function *() {
+    yield ember([
+      'new',
+      'foo',
+      '--skip-npm',
+      '--skip-bower',
+      '--skip-git',
+    ]);
+
+    expect(file('package.json'))
+      .to.match(/"ember-welcome-page"/);
+
+    expect(file('app/templates/application.hbs'))
+      .to.contain(/"{{welcome-page}}"/);
```

Comment:
I *think* the issue is that you wrote `/"{{welcome-page}}"/` instead of `/{{welcome-page}}/`. Could you try this without the quotes? or just use `expect(...).to.contain('{{welcome-page}}');` instead.